### PR TITLE
Pull Request for Mike

### PR DESCRIPTION
**Pull request by shannahutch**
First PR for mike to comment on.

### CODE REVIEW

File: react.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  
+<head>
+  <title>1React-app</title>
+  <link rel="stylesheet" type="text/css" href="theme.css">
+  <script src="https://fb.me/react-15.1.0.js"></script>
+  <script src="https://fb.me/react-dom-15.1.0.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-core/5.8.23/browser.min.js"></script>
+  <script src="https://npmcdn.com/react-router@2.4.0/umd/ReactRouter.min.js"></script>
+
+  <head>
```
Comment (mikemintz) [inline react.html]: I think you mean `</head>`

File: react.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  					);
+  				}
+  			});
+
+  			var Stuff = React.createClass({
+  				render: function() {
+  					return (
+  						<div>
+  							<h2>Stuff</h2>
+  							<p>Turltes</p>
```
Comment (mikemintz) [inline react.html]: That's not how to spell turtles!

File: react.html [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  							<ol>
+						        <li>1 turtle</li>
+						        <li>2 turtle</li>
+						        <li>3 turtle</li>
+						        <li>4 turtle</li>
+					            <li>More turtle</li>
```
Comment (mikemintz) [inline react.html:54]: Typically in react, you would have 10 turtles before generalizing "more turtle"

**Comment by mikemintz** [comment]
Looks great to me.

I'd recommend using 2 spaces instead of tabs for indentation, since that seems pretty much standard these days in JS.

I'd also recommend using `const` instead of `var` in the places you have. You have to use `var` with ES5 (older javascript syntax), but since you're using babel anyway here, you may as well use `const` to keep your code more safe. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const

File: react.html (hunk 2 above)
Comment (shannahutch) [inline react.html]: veru true

File: react.html (hunk 3 above)
Comment (shannahutch) [inline react.html:54]: Tell me you are messing with me~

Comment (mikemintz) [inline react.html:54]: Well it is more of a turtle best practice than a react best practice. It just seems weird that you would only have 4 turtles, when you could have a lot more, you know?

**Pull request merged**